### ISSUE TITLE
key contained in another key

### ISSUE THREAD

**Issue by riverinyo** [Respect/Validation#165]
I'm validating POST.
I have an input name="phone" and another input name="phoneXXXX"
when the second one fails the test, makes the first one fail as well.
It looks like the problem is the name contained inside the second one, cause changing my input name, make it work fine.
Thanks, guys.

**Comment by alganet** [comment]
I'm pretty sure the problem is key conflict in the form. Can you show us a `print_r($_POST)`?

**Comment by wesleyvicthor** [comment]
@riverinyo any news ?

**Comment by riverinyo** [comment]
Hey,
Sorry for the delay and thanks for your support.
That happen long ago and the project has changed a bit since then.
I don't think it make sense to fix it. Yet, I tried to search back in the repo to see how did I solve it, but it looks like we discarded the other fields.
This is what I have in my code (commented partially, though)
Validator::arr()
->key('phone', Validator::phone()->notEmpty())
->key('phoneNetwork', Validator::digit()->notEmpty()->noWhitespace()->length(10,10)
...
Sorry and thanks again.
